guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -385,9 +375,46 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
         return adtIcon;
     }
 
-
     render() {
 
+        if(this.props.globalState.guiTexture == undefined) return null;
+        const _sizeValues = [
+            new Vector2(1920, 1080),
+            new Vector2(1366, 768),
+            new Vector2(1280, 800),
+            new Vector2(3840, 2160),
+            new Vector2(750, 1334),
+            new Vector2(1125, 2436),
+            new Vector2(1170, 2532),
+            new Vector2(1284, 2778),
+            new Vector2(1080, 2220),
+            new Vector2(1080, 2340),
+            new Vector2(1024, 1024),
+            new Vector2(2048, 2048),
+        ];
+        const _sizeOptions = [
+            { label: "Web (1920)", value: 0 },

```

Comment:
so, i wonder - why reference the value as an index in the array when the values are not being reused? Why not keep the vector2 here as value?

Reply:
It's used lower? I'm not sure what this question means.

Reply:
wh is value not `Vector2(102, 1024)` ? Why do you need the other array if you already have a 1-to-1 map?

Reply:
Discussed offline. part of the structure of optionsLineComponent.